src/p24/Przelewy24.ts

```diff
@@ -137,13 +137,18 @@ export class Przelewy24 {
      */
     public async verifyTransaction (verification: TransactionVerification) {
         const crcData = `${verification.p24_session_id}|${verification.p24_order_id}|${verification.p24_amount}|${verification.p24_currency}|${this.salt}`
+
         const hash = crypto.createHash('md5')
             .update(crcData)
             .digest('hex');
+
+        if (verification.p24_sign !== hash) {
```

Comment:
this verification happens from p24 side, p24 verifies the crcValue with their backend